packages/nala/build.sh

```diff
@@ -32,42 +26,19 @@ termux_step_pre_configure() {
 		${_CROSSENV_PREFIX}
 	popd
 	. ${_CROSSENV_PREFIX}/bin/activate
-
-	build-pip install tomli
 }
 
 termux_step_make_install() {
-	python setup.py install --force
-
 	export PYTHONPATH=$TERMUX_PREFIX/lib/python${_PYTHON_VERSION}/site-packages
-	python setup.py install --force --prefix $TERMUX_PREFIX
-
-	pushd $PYTHONPATH
-	_NALA_EGG=
-	for f in nala-${TERMUX_PKG_VERSION##*:}-py${_PYTHON_VERSION}.egg; do
-		if [ -e "$f" ]; then
-			_NALA_EGG="$f"
-			break
-		fi
-	done
-	test -n "${_NALA_EGG}"
-	popd
+	python -m pip install . --force --prefix $TERMUX_PREFIX
 
 	install -Dm600 -t $TERMUX_PREFIX/etc/nala debian/nala.conf
 }
 
 termux_step_create_debscripts() {
 	cat <<- EOF > ./postinst
 	#!$TERMUX_PREFIX/bin/sh
-	echo "./${_NALA_EGG}" >> $TERMUX_PREFIX/lib/python${_PYTHON_VERSION}/site-packages/easy-install.pth
 	mkdir -p $TERMUX_PREFIX/var/lib/nala
 	mkdir -p $TERMUX_PREFIX/var/log/nala
```

Comment:
```suggestion
	mkdir -p $TERMUX_PREFIX/var/log/nala
	mkdir -p $TERMUX_PREFIX/var/lock
```
Adresses crash described [here](https://github.com/termux/termux-packages/pull/11843#issuecomment-1244690489) related to `$TERMUX_PREFIX/var/lock/` (potentially) missing, thus being unable to create a lock file.